contrib/FreeBSD/README.md

```diff
@@ -0,0 +1,58 @@
+This folder contains `sopel` and `sopel-default.cfg` designed to be distributed by third parties such as FreeBSD.
+
+`sopel-default.cfg` is a default configuration file for sopel.
+
+The -default prefix is mandatory and will be used as a 'profile'.
+
+The profiles may be changed with the variable `sopel_profiles` in `/etc/rc.conf`. For example:
+
+```sh
+sysrc sopel_profiles="profile1 profile2 profile3"
+```
+
+By default, the configuration directory is `/usr/local/etc`.
+
+The `sopel-profile1.cfg`, `sopel-profile2.cfg` and `sopel-profile3.cfg` files must be stored in the configuration directory.
```

Comment:
```suggestion
For the above example, the configuration files `sopel-profile1.cfg`, `sopel-profile2.cfg`, and `sopel-profile3.cfg` must be present there.
```
Conditional on the example - otherwise I can see someone reading it as mandatory.